projects/plugins/jetpack/modules/tiled-gallery/tiled-gallery/tiled-gallery-shape.php

```diff
@@ -12,6 +12,20 @@ class Jetpack_Tiled_Gallery_Shape {
 	 */
 	public static $shapes_used = array();
 
+	/**
+	 * The images to include in gallery.
+	 *
+	 * @var object
```

Comment:
As with some others above, looks like both this and the constructor parameter should actually be `object[]`.

Reply:
Done in b503fca7d2b44b8c41de15263072a4145cc1b69c